.github/workflows/build-ubuntu.yaml

```
@@ -105,11 +113,29 @@ jobs:
       - name: Install cached non packaged dependencies
         if: steps.build-cache-restore-step.outputs.cache-hit != 'true' # Variable type is string, thus using quotes
         working-directory: deps
-        run: ../docker/build_deps.sh
+        run: |
+          ${{ inputs.cpp03 }} ]] && ../docker/build_deps.sh --cpp03 || ../docker/build_deps.sh
```

```suggestion
          [[ ${{ inputs.cpp03 }} ]] && ../docker/build_deps.sh --cpp03 || ../docker/build_deps.sh
```
Don't understand how it worked before